lib/batch.js

```diff
@@ -12,7 +5,7 @@ var debug = require('debug')('wpcom:batch');
  * @api public
  */
 
-function Batch(wpcom) {
+function Batch(wpcom) { //eslint-disable-line space-before-function-paren
```

Comment:
Maybe we out to be declaring the rule as:

`"space-before-function-parent": { "anonymous": "always", "named": "never" }`

http://eslint.org/docs/rules/space-before-function-paren